### ISSUE TITLE
[feature]: add RPC (and lncli command) to retrieve funding transaction

### ISSUE THREAD

**Issue by Kixunil** [lightningnetwork/lnd#6984]
**Is your feature request related to a problem? Please describe.**

In a test environment I encountered a situation when the funding transaction didn't broadcast. I'm not sure why but it could theoretically happen on mainnet too it'd be nice to have a way to retrieve the transaction and broadcast it manually.

**Describe the solution you'd like**
Have something like `getfundingtransaction <channel-id>` command in `lncli` and a corresponding 

**Describe alternatives you've considered**
Rebroadcast periodically or rebroadcast triggered by RPC command. These are fine solutions but being able to manually rebroadcast is probably nicer for debugging (one can analyze transaction or try out different broadcasting methods).

**Comment by guggero** [comment]
I don't think this requires any RPC changes. The `lncli listchaintxns` should list all transactions and can be matched by the channel funding outpoint. That output has the full raw transaction bytes that can then be broadcast again using `lncli wallet publishtx`. Or am I missing something?

**Comment by Kixunil** [comment]
Ah, missed it, thanks! I wonder if it could be made more discoverable. Nothing particularly great comes to my mind. Maybe this issue existing will be good enough - people may search for it.
